areas/areas.go

```diff
@@ -0,0 +1,162 @@
+package areas
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/ONSdigital/dp-api-clients-go/v2/clientlog"
+	"github.com/ONSdigital/dp-api-clients-go/v2/headers"
+	healthcheck "github.com/ONSdigital/dp-api-clients-go/v2/health"
+	health "github.com/ONSdigital/dp-healthcheck/healthcheck"
+	dprequest "github.com/ONSdigital/dp-net/request"
+	"github.com/ONSdigital/log.go/v2/log"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+)
+
+const service = "areas-api"
+
+// ErrInvalidAreaAPIResponse is returned when the area api does not respond
+// with a valid status
+type ErrInvalidAreaAPIResponse struct {
+	actualCode int
+	uri        string
+	body       string
+}
+
+// Error should be called by the user to print out the stringified version of the error
+func (e ErrInvalidAreaAPIResponse) Error() string {
+	return fmt.Sprintf("invalid response: %d from dataset api: %s, body: %s",
+		e.actualCode,
+		e.uri,
+		e.body,
+	)
+}
+
+// Code returns the status code received from Area api if an error is returned
+func (e ErrInvalidAreaAPIResponse) Code() int {
+	return e.actualCode
+}
+
+// Client is a areas api client which can be used to make requests to the server
+type Client struct {
+	hcCli *healthcheck.Client
+}
+
+// New creates a new instance of Client with a given areas api url
+func New(areasAPIURL string) *Client {
+	return &Client{
+		healthcheck.NewClient(service, areasAPIURL),
+	}
+}
+
+// NewWithHealthClient creates a new instance of Client,
+// reusing the URL and Clienter from the provided health check client.
+func NewWithHealthClient(hcCli *healthcheck.Client) *Client {
+	return &Client{
+		healthcheck.NewClientWithClienter(service, hcCli.URL, hcCli.Client),
+	}
+}
+
+// Checker calls areas api health endpoint and returns a check object to the caller.
+func (c *Client) Checker(ctx context.Context, check *health.CheckState) error {
+	return c.hcCli.Checker(ctx, check)
+}
+
+// GetArea returns area information for a given area ID
+func (c *Client) GetArea(ctx context.Context, userAuthToken, serviceAuthToken, collectionID, areaID string) (areaDetails AreaDetails, err error) {
+	uri := fmt.Sprintf("%s/areas/%s", c.hcCli.URL, areaID)
```

Comment:
This will have to be updated to:

`uri := fmt.Sprintf("%s/v1/areas/%s", c.hcCli.URL, areaID)